pycloudflare/utils.py

```diff
@@ -0,0 +1,17 @@
+def _find_server_error(response, err_code=None, msg_part=None):
```

Comment:
I'd be tempted to turn this into a wrapper that turns an error code into an exception class.